critiquebrainz/frontend/external/musicbrainz_db/utils.py

```diff
@@ -0,0 +1,17 @@
+import critiquebrainz.frontend.external.musicbrainz_db.exceptions as mb_exceptions
+
+
+def get_entities_by_gids(query, entity_model, redirect_model, mbids):
```

Comment:
Can you please add a docstring describing what this function does and what the arguments are (can use type annotations to indicate types as well)?